projects/packages/action-bar/src/scss/action-bar.scss

```diff
@@ -17,6 +37,602 @@
 	background: none;
 	border: none;
 	line-height: 0;
-	margin: 8px;
+	padding: 8px;
+	margin: 0;
+}
+/* .action-bar {
```

Comment:
I left in some commented out css because we are basing our CSS on work @davemart-in did in an HMTL prototype 2dcb4-pb, it will be enabled as we add the features it corresponds to.